src/makeCalculator.js

```diff
@@ -4,7 +4,28 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (num) => calculator.result + num,
```

Comment:
[CODE KNOWLEDGE] You are currently returning the result of the operation instead of modifying `calculator.result`. It means that `calculator.result` stays unchanged after calling `add`, `subtract`, `multiply`, `divide` methods. You should use assignment operator `=` to update the `calculator.result` in each of these methods.